src/components/cuda/cupti_common.c

```diff
@@ -248,6 +258,10 @@ static int util_load_cuda_sym(void)
     papi_errno = load_cuda_sym();
     papi_errno += load_cudart_sym();
     papi_errno += load_cupti_common_sym();
```

Comment:
Unrelated to this PR. Please never do something like this. Always handle one function at a time.